<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(158, 202, 225);
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:rgb(8, 48, 107);
font: 75 12pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>Change Old Password</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(66, 146, 198);
border-radius:20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);
color:rgb(3, 37, 57);
font: 75 10pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Unique Token&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="newPassword_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);
color:rgb(3, 37, 57);
font: 75 10pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;New Password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="newPassword_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);
color:rgb(3, 37, 57);
font: 75 10pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Confirm Password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="uniqueToken">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);;
color:rgb(3, 37, 57);
font: 75 10pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newPassword">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);;
color:rgb(3, 37, 57);
font: 75 10pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmPassword">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 243, 242);;
color:rgb(3, 37, 57);
font: 75 10pt &quot;Verdana&quot;;
border: 1px solid rgb(28, 118, 143);
border-radius: 8px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verifyBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(28, 118, 143);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 8px;
color:rgb(251, 243, 242);
background-color:rgb(85, 170, 127)</string>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="togglePasswordCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(28, 118, 143);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 8px;
color:rgb(251, 243, 242);
background-color:rgb(85, 170, 127)</string>
    </property>
    <property name="text">
     <string>Show password</string>
    </property>
   </widget>
   <zorder>line</zorder>
   <zorder>label_4</zorder>
   <zorder>newPassword_2</zorder>
   <zorder>newPassword_4</zorder>
   <zorder>uniqueToken</zorder>
   <zorder>newPassword</zorder>
   <zorder>confirmPassword</zorder>
   <zorder>verifyBtn</zorder>
   <zorder>togglePasswordCheckbox</zorder>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:rgb(8, 48, 107);
font: 75 12pt &quot;Verdana&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;Please enter your favorite food first to change your old password.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(28, 118, 143);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 8px;
color:rgb(251, 243, 242);
background-color:rgb(255, 87, 87);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(28, 118, 143);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-radius: 8px;
color:rgb(251, 243, 242);
background-color:rgb(33, 113, 181);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
